src/components/button/Button.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withKnobs, text, select, object } from '@storybook/addon-knobs'
+import { action } from '@storybook/addon-actions'
+import styled from 'styled-components'
+import { variant } from 'styled-system'
+import { colors } from '../../theme/colors'
+import { Button } from './Button'
+import { ButtonBaseProps } from './Button.types'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of file extension "types" for "./Button.types"](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)